core/frontend/src/render/FrameStats.ts

```diff
@@ -0,0 +1,141 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module Rendering
+ */
+
+import { BeTimePoint } from "@bentley/bentleyjs-core";
+
+/** Describes timing statistics for a single rendered frame.

```

Comment:
Explain how frames are not rendered continuously by default and therefore, e.g., `totalFrameTime` cannot be used to compute FPS.